store/ZkLobbyServer.js

```diff
@@ -456,6 +464,27 @@ module.exports = function(){ return Reflux.createStore({
 			this.sentLaunchRequest = false;
 			
 		},
+		
+		// match maeking
+		"MatchMakerSetup": function(msg){
+			MatchMaking.queues = msg.PossibleQueues;
```

Comment:
I'm not sure what the point of the separate MM store is. It doesn't do anything and can be replaced by an object in ZkLobbyServer state.

Also this goes against the canon because the canon way is to have MM store subscribe to LobbyServer and react to changes in it (and then you'll have an object with ```queues``` and ```activeQueues``` anyway in ZkLobbyServer, making the MM store superfluous).

And it's a singleton, all stores used to be singletons in the past, then I made them explicitly created for zkwl, it's probably a good idea to keep that for cleanness if nothing else.